website/src/components/pages/single-component/blocks-hub.js

```diff
@@ -12,6 +12,12 @@ const Bold = props => <strong css={{ fontWeight: 'bold' }} {...props} />;
 const Italic = props => <em css={{ fontStyle: 'italic' }} {...props} />;
 const Strike = props => <span css={{ textDecoration: 'strike-through' }} {...props} />;
 const Under = props => <span css={{ textDecoration: 'underline' }} {...props} />;
+const leftIndent = {
+	paddingLeft: '5rem',
+	'@media (max-width: 568px)': {
```

Comment:
Are breakpoints in the design tokens?

Reply:
Yes but that doesn't mean you can add breakpoints anywhere where it makes sense